test/functional/ui/screen_basic_spec.lua

```diff
@@ -690,4 +689,12 @@ describe('Screen', function()
       end)
     end)
   end)
+
+  it('nvim_ui_attach() handles very large width/height #2180', function()
+    screen:detach()
+    screen = Screen.new(999, 999)
```

Comment:
If the test isn't sensitive to timing, but just takes longer to run, there's no need to mark it.